server.js

```diff
@@ -0,0 +1,160 @@
+const express = require('express');
+const app = express();
+const bp = require('body-parser');
+const exphbs = require('express-handlebars');
+const methodOverride = require('method-override')
+const Products = require('./dataBase/products.js');
+const Articles = require('./dataBase/articles.js');
+
+
+//this is the inventory class to store items pulled from the database file
+const DS_Inv = new Products();
+const DS_Inv2 = new Articles();
+
+app.use(express.static('public'));
+app.use(bp.urlencoded({ extended: true }));
+app.use(methodOverride('_method'));
+
+app.engine('.hbs', exphbs({ defaultLayout: 'main', extname: '.hbs' }));
+app.set('view engine', '.hbs');
+
+
+app.get('/', (req, res) => {
+  res.render('home')
+});
+
+
+//~~~~~~~~~~~~~~ PRODUCTS ~~~~~~~~~~~~~~//
+
+//this will render all the products
+app.get('/products', (req, res) => {
+  const items = DS_Inv.all();
+  console.log('products: ', items);
+  res.render('products', { items });
+})
+
+//this will render out the form to get the products to add to the list    
+app.get('/products/new', (req, res) => {
+  res.render('add-products-form');
+})
+
+//this will render out the detail of the products by the id of the item
+app.get('/products/:id', (req, res) => {
+  const { id } = req.params;
```

Comment:
`let`, is a signal that the variable may be reassigned, such as a counter in a loop, or a value swap in an algorithm. It also signals that the variable will be used only in the block it’s defined in, which is not always the entire containing function. https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75 - change these uses of const to 'let', as id will change and let implies local, const is misleading.